src/http-api/routes/bootstrap.js

```diff
@@ -17,14 +16,11 @@ module.exports = (server) => {
   api.route({
     method: '*',
     path: '/api/v0/bootstrap/add',
-    handler: resources.bootstrap.add,
     config: {
-      validate: {
-        query: {
-          arg: Joi.string().required(), // multiaddr to add
-          default: Joi.boolean()
-        }
```

Comment:
I don't know why, but it wouldn't allow for any additional query parameters to be passed along except for the ones that were part of this validation.